src/sympc/protocol/fss/fss.py

```diff
@@ -180,6 +180,8 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
 class FSS(metaclass=Protocol):
     """Function Secret Sharing."""
```

Comment:
Add the purpose of the share class to the FSS docstring that describes what it does.